### PR TITLE
add all Opel vehicles to common.js #59

### PR DESCRIPTION
**Pull request by sennahLessert**
i added all opel models from Apendix A

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Opel': {
+      '1900': ['fsp'],
```
Comment (Bjorn248) [inline common.js:1947]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

File: common.js [hunk 2]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
...
+        '1975': ['dp']
+      },
+      'Ascona SportWagon (1900cc)': {
+        '1971': ['dp'],
+        '1972': ['dp'],
+        '1973': ['dp'],
+        '1974': ['dp'],
+        '1975': ['dp']
+      },
+      'Manta': ['fsp'],
```
Comment (Bjorn248) [inline common.js:1962]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

File: common.js [hunk 3]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
...
+        '1975': ['dp']
+      },
+      'Manta Rallye (1900 cc)': {
+        '1971': ['dp'],
+        '1972': ['dp'],
+        '1973': ['dp'],
+        '1974': ['dp'],
+        '1975': ['dp']
+      },
+      'GT 1100': ['fsp', 'dp'],
```
Comment (Bjorn248) [inline common.js:1977]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

File: common.js [hunk 4]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
...
+      'GT 1500': ['fsp'],
```
Comment (Bjorn248) [inline common.js:1978]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

File: common.js [hunk 5]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
...
+      'GT 1900': ['fsp', 'dp'],
```
Comment (Bjorn248) [inline common.js:1979]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

File: common.js [hunk 6]
```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
...
+          '1965': ['fsp', 'dp'],
+          '1966': ['fsp', 'dp'],
+          '1967': ['fsp', 'dp'],
+          '1968': ['fsp', 'dp'],
+          '1969': ['fsp', 'dp'],
+          '1970': ['fsp', 'dp'],
+          '1971': ['fsp', 'dp'],
+          '1972': ['fsp', 'dp']
+      },
+      'Kadett 1500': ['fsp'],
```
Comment (Bjorn248) [inline common.js:1991]: This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Inconsistent Object Structure, needs to be fixed before I can merge.

File: common.js (hunk 1 above)
Comment (sennahLessert) [reply common.js:1947]: Ok i'll fix that

**Comment by sennahLessert** [comment]
i changed the structure

File: common.js [hunk 7]
```diff
@@ -2545,6 +2545,7 @@ const allSoloCars = {
       '2004': ['ep'],
     },
   },
+<<<<<<< HEAD
```
Comment (Bjorn248) [inline common.js:2548]: This looks like some leftover text from a merge conflict?

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Need to remove merge conflict text

**Comment by sennahLessert** [comment]
sorry for the mistakes. I think now the structure is fine and all merge conflicts were resolved

**Pull request merged**